caddy_chatbot/src/caddy_core/components.py

```diff
@@ -1,37 +1,34 @@
+import json
+import os
 from datetime import datetime
-from fastapi.responses import Response
-from fastapi import status
-
-from langchain.prompts import PromptTemplate
-from caddy_core.utils.prompt import retrieve_route_specific_augmentation, get_prompt
-
 from time import sleep
+from typing import Any, Dict, List, Tuple
 
+from boto3.dynamodb.conditions import Key
 from caddy_core.models import (
-    ProcessChatMessageEvent,
+    ApprovalEvent,
     CaddyMessageEvent,
-    UserMessage,
     LlmResponse,
+    ProcessChatMessageEvent,
     SupervisionEvent,
-    ApprovalEvent,
+    UserMessage,
 )
-
+from caddy_core.services import enrolment
+from caddy_core.services.evaluation import execute_optional_modules
+from caddy_core.services.retrieval_chain import build_chain
+from caddy_core.utils.monitoring import logger
+from caddy_core.utils.prompt import get_prompt, retrieve_route_specific_augmentation
 from caddy_core.utils.tables import (
     evaluation_table,
     responses_table,
     users_table,
 )
-from caddy_core.utils.monitoring import logger
-from caddy_core.services.retrieval_chain import build_chain
-from caddy_core.services import enrolment
-from caddy_core.services.evaluation import execute_optional_modules
-from boto3.dynamodb.conditions import Key
-
-import json
+from fastapi import status
+from fastapi.responses import Response
+from langchain.prompts import PromptTemplate
+from langchain_community.chat_models import BedrockChat
```

Comment:
We may want to update this import for BedrockChat as it's deprecated 

"Deprecated since version 0.0.34: Use langchain_aws.ChatBedrock instead." 

We already have langchain_aws as a dependency so it's just a case of swapping out to  `from langchain_aws import ChatBedrock`